reset.d.ts

```diff
@@ -0,0 +1 @@
+import "@total-typescript/ts-reset";
```

Comment:
really cool little lib that fixes a lot of ts headaches https://github.com/total-typescript/ts-reset